ONSdigital/dp-frontend-router
# Feature/search stats jwt

### What

Use a JWT for `/redir` value

### How to review

* Start dp-frontend-router
* Start babbage on `feature/search-stats-jwt` branch
* Click a search result and redirect should be successful
* Try opening `http://localhost:20100/redir/some-fake-value` - should get an error

### Who can review

Anyone except @ian-kent
